binscripts/rvm-installer

```diff
@@ -205,7 +205,7 @@ __rvm_curl()
   }
 
   typeset -a __flags
-  __flags=( --fail --location --max-redirs 10 )
+  __flags=( -q --fail --location --max-redirs 10 )
```

Comment:
I would rather go with:
```suggestion
  __flags=( $RVM_CURL_FLAGS --fail --location --max-redirs 10 )
```
this way you can overwrite any setting you like, but in most cases the `.curlrc` will be what users need like proxies, in case users have some nonstandard settings they can overwrite it.